demos/rag_agentic/notebooks/Level2_simple_agentic_with_websearch.ipynb

```diff
@@ -176,15 +162,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3. Running the agent.\n",
-    "- Define user prompts to interact with the agent.\n",
-    "- Use the agent to create a session and process user queries.\n",
-    "- Display the agent's responses for each query."
+    "## 3. Run the agent.\n",
+    "- Populate `user_prompts\" with questions that you would like to ask the agent.\n",
+    "- Create a unique agent session for this conversation so that it can store metadata and context history in the Llama Stack server.\n",
+    "- Finally, display the agent's responses for each query."
```

Comment:
so in order to output every step of the agent (as we have in this rendered notebook) we actually need to set `STREAM=False`. Should we add a note here to mention that? else it might be confusing to the user why they aren't seeing outputs rendered for each step and also maybe its better to set the default to `False` instead of `True` in the level0 notebook

Reply:
Setting default to false